modules/nf-core/gawk/meta.yml

```diff
@@ -0,0 +1,46 @@
+name: "gawk"
+description: If you are like many computer users, you would frequently like to make changes in various text files wherever certain patterns appear, or extract data from parts of certain lines while discarding the rest. To write a program to do this in a language such as C or Pascal is a time-consuming inconvenience that may take many lines of code. The job is easy with awk, especially the GNU implementation gawk.
```

Comment:
Rather long, I would cut this down

Reply:
Oops copy pasted from the official source, I'll shorten it :)